toolkit/docs/building/building.md

```diff
@@ -95,87 +103,48 @@ cd CBL-Mariner/toolkit
+The toolchain builds in two sub-phases.  The first phase builds an initial _bootstrap_ toolchain which is then used to build the _final_ toolchain used in package building.  In the first phase, the bootstrap toolchain downloads a series of source packages from upstream sources.  The second phase downloads SRPMS from packages.microsoft.com.
 
-```bash
-# Download the pre-built toolchain packages (REBUILD_TOOLCHAIN=n is the default value)
-sudo make toolchain REBUILD_TOOLS=y
+For expediency, the toolchain may be populated from upstream binaries, or may be completely rebuilt. 
 
-# Move the downloaded bootstrap packages to the general RPM out folder
-sudo make copy-toolchain-rpms
-```
+### **Populate Toolchain**
 
-NOTE: A full list of targets and options to `make` is available [here](#all-build-targets) and [here](#all-build-variables).
+A set of bootstrapped toolchain packages (gcc etc.) are used to build CBL-Mariner packages and images.  Rather than built the toolchain, the prebuilt binaries can be downloaded to your local machine.
 
-##### Full toolchain Bootstrap From Sources
+### **Rebuild Toolchain**
 
-If you want to build **everything** from scratch, including the bootstrapping process, run:
+Depending on hardware, rebuilding the toolchain can take several hours. The following builds **the entire toolchain** from scratch:
 
 ```bash
-cd ~/git/CBL-Mariner/toolkit
-
-# Do a FULL bootstrap + rebuild from sources instead (much slower)
 # Add REBUILD_TOOLCHAIN=y to any subsequent command to ensure locally built toolchain packages are used
 sudo make toolchain REBUILD_TOOLS=y REBUILD_TOOLCHAIN=y DOWNLOAD_SRPMS=y PACKAGE_IGNORE_LIST="openjdk8 openjdk8_aarch64 shim-unsigned-aarch64"
 ```
 
-This will download the source files (SRPMs) from the package sever, and build them locally.
-See the detailed section on building from scratch [here](#building-everything-from-scratch)
-
-#### 2) Build All Packages
+## **Package Stage**
 
-(**this step may be omitted if desired**)
+After the toolchain is built or populated, package building is possible.The CBL-Mariner ecosystem provides a significant number of packages, but most of those packages are not used in an image.  When rebuilding packages, you can choose to build everything, or you can choose to build just what you need for a specific image.  This can save significant time because only the subset of the CBL-Mariner packages needed for an image are built.
 
-Once the toolchain is bootstrapped, packages can be built with the toolchain.
+The CONFIG_FILE argument provides a quick way to declare what to build. To manually build **all** packages you can clear the configuration with `CONFIG_FILE=` and invoke the package build target.  To build packages needed for a specific image, you must set the CONFIG_FILE= parameter to an image configuration file of your choice.  The standard image configuration files are in the toolkit/imageconfigs folder.  
```

Comment:
>standard [](start = 336, length = 8)

Suggestion: I'm not sure if "sample" wouldn't describe these configs better than "standard".

Reply:
I think I prefer standard, since these are in fact the standard set of image configurations we create out of Mariner Core.